public/scripts/scripts.js

```diff
@@ -42,25 +40,26 @@ function getPlaylistTracks(access_token, allCallSongs, request_url, playListTrac
       console.log(tracks);
       tracks.map(function(track){
         playListTracks.push(track);
-        writePlayListToPanel(track, playListTracks);
```

Comment:
i noticed that writePlayListToPanel was always called in a 

```
playListTracks.map(function(track) {
    writePlayListToPanel(track);
})
```
loop (this occurs throughout the diff), so i figured it'd be simpler to just make the function take `playListTracks` and have the function be responsible for doing that `.map()` call itself. this makes things easier on callers of the function, and prevents us from calling `initTrackListener()` too many times.

Reply:
Makes sense!